src/deploy/functions/checkIam.ts

```diff
@@ -148,7 +148,7 @@ export function mergeBindings(policy: iam.Policy, allRequiredBindings: iam.Bindi
  * @param have backend that we have currently deployed
  */
 export async function ensureServiceAgentRoles(
-  projectId: string,
+  projectNumber: string,
```

Comment:
per lint, please update javadoc to match the new param name.

Reply:
This was already updated in a subsequent PR, master has the correct tsdoc